### ISSUE TITLE
foundation_paginate doesn't support parameters of will_paginate

### ISSUE THREAD

**Issue by simo2409** [markmcconachie/foundation-will_paginate#1]
For example if you try to customize pagination param with:

<%= foundation_paginate @users, :param_name => :user_page %>

You get an error while it works with will_paginate:

<%= will_paginate @users, :param_name => :user_page %>

**Comment by acrogenesis** [comment]
@simo2409 did you find a solution for this?

**Comment by simo2409** [comment]
@acrogenesis No, they are just "not supported" by the gem, I didn't write code because I was only trying it

**Comment by acrogenesis** [comment]
@simo2409 if you are still looking for a solution I found one here is the gist, just add this to your application helper
https://gist.github.com/acrogenesis/6731908
just add the params to the will_paginate_foundation and use on your view
<%= will_paginate_foundation(@pages) %>

**Comment by simo2409** [comment]
@acrogenesis Interesting, thank you for sharing :+1:

**Comment by acrogenesis** [comment]
For anyone who finds this problem i made a new gem which works great for adding params and other things https://github.com/acrogenesis/will_paginate-foundation
